### ISSUE TITLE
Edit mode slider has no visible focus state

### ISSUE THREAD

**Issue by pbakaus** [GoogleForCreators/web-stories-wp#1425]
## Bug Description

The edit mode slider has no visible focus state. This is important for accessibility.

![image](https://user-images.githubusercontent.com/43004/80780577-5f2ba200-8b24-11ea-875a-eb54c925a95e.png)

## Expected Behaviour

There should be a visible focus state when the slider is in focus.

## Screenshots

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## QA Instructions

_See #3883_

**Comment by jauyong** [comment]
@samitron7 I added UX Need here. I know we have [this section in figma](https://www.figma.com/file/Cd9iacVknHK62RO4pRwb68/Stories-Editor?node-id=9396%3A400304), but does it apply to the edit mode slider as well?

**Comment by wassgha** [comment]
@samitron7 any update on this?

**Comment by samitron7** [comment]
I would like for us to gather up all the focus related issues and prioritize those so I can holistically view and work through them all. (Prioritization is needed here for me because I won't be able to get to this right now @chris-pgm.

**Comment by swissspidy** [comment]
This should be moved out of the sprint and into the backlog.

An epic could be created to cover this more holistically.

**Comment by miina** [comment]
Adding "Needs more Info" status -- if I understand correctly from the comments then this still needs looking into for having specific Acceptance Criteria.

**Comment by jauyong** [comment]
flagging @o-fernandez @bmattb for awareness on more info needed.

**Comment by samitron7** [comment]
I believe @pbakaus is going to take on accessibility work and help me prioritize as well (?)

**Comment by pbakaus** [comment]
@samitron7 small correction: I won't be able to take on all a11y work, but I am [writing guidelines](https://docs.google.com/document/d/15FHCukTDrWf7xYpPkBG3bGRgRQZeqzkCcglaNXgaw9s/edit#) and am happy to coordinate. Eng and QA still needs to help track and implement fixes. 

/cc @dvoytenko, who might be the best candidate to look at doing a pass around keyboard nav and focus states
